chaoscenter/graphql/server/graph/chaos_experiment_run.resolvers.go

```diff
@@ -53,6 +52,24 @@ func (r *mutationResolver) RunChaosExperiment(ctx context.Context, experimentID
 	return &model.RunChaosExperimentResponse{NotifyID: uiResponse.NotifyID}, err
 }
 
+func (r *mutationResolver) StopExperimentRuns(ctx context.Context, projectID string, experimentID string, experimentRunID *string, notifyID *string) (bool, error) {
+	logFields := logrus.Fields{
+		"projectId":            projectID,
+		"chaosExperimentId":    experimentID,
+		"chaosExperimentRunId": experimentRunID,
+		"notifyID":             notifyID,
+	}
+
+	logrus.WithFields(logFields).Info("request received to stop chaos experiment")
+	err := authorization.ValidateRole(ctx, projectID,
+		authorization.MutationRbacRules[authorization.StopChaosExperiment],
+		model.InvitationAccepted.String())
+	if err != nil {
+		return false, err
+	}
+
```

Comment:
I think the call to the actual StopExperimentRuns is missing here?

Reply:
Yes it was in WIP, have added the implementation.